change/@fluentui-react-slider-fb843414-dfa0-400c-bb4d-eab3a319d5ad.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "prerelease",
+  "comment": "Move Slider to RC",
+  "packageName": "@fluentui/react-slider",
```

Comment:
Stories are dev dependencies and not actually published. In this case the message for this changefile should be about the css variable changes you made in the component